### PR TITLE
Issue #210

### PR DESCRIPTION
**Pull request by daft300punk**

### CODE REVIEW

File: src/shared/components/challenge-listing/index.jsx [hunk 1]
```diff
@@ -85,6 +85,12 @@ export default function ChallengeListing(props) {
     );
   }
 
+  /* eslint-env browser */
+  if (document) {
```
Comment (birdofpreyru) [inline src/shared/components/challenge-listing/index.jsx]: @daft300punk To check, whether the code is executed at server or client side, please always use `isClientSide()` or `isServerSide()` from https://github.com/topcoder-platform/community-app/blob/develop/src/shared/utils/isomorphy.js

Also, placing this code into render method, I expect, will have side effects: ReactJS is free to re-render a component in many cases, not only when we enter the page, thus this code, I believe, will scroll the page to top in some cases when we don't need it. I guess, we should put it into `onClick` handler of the `View more challenges` button. Also, inside that handler you don't have to check for client side, as click handlers are always invoked at client-side only.

**Comment by birdofpreyru** [comment]
@daft300punk any update to this?

**Comment by birdofpreyru** [comment]
@daft300punk Hey-ho? Any update?!!

**Comment by daft300punk** [comment]
Checkout now. Sorry for the delay, was a little away for visa purposes.

**Pull request merged**